### PR TITLE
Set up ReadTheDocs

### PR DESCRIPTION
**Pull request by atedstone**
ReadTheDocs is now working for this branch (https://geoutils.readthedocs.io/en/docs/) - these commits contain the eventual working configuration.

Also note that requirements.txt now added

### CODE REVIEW

**Comment by atedstone** [comment]
Fixes #62 .

**Comment by adehecq** [comment]
Very basic, but looks like a good start! Will you be able to add the API?

**Comment by atedstone** [comment]
Pfft, you have no idea the amount of grief it's taken just to get to this point....thanks @jlandmann for help too!

But yes I will try to get an API page working. 

We should also start user guides/tutorials for georaster and geovector.

File: setup.py [hunk 1]
```diff
@@ -37,3 +25,18 @@ def write_version_py(filename=None):
...
+
+
+setup(name='geoutils',
+      version=FULLVERSION,
+      description='',
+      url='',
+      author='The GlacioHack Team',
+      license='BSD-3',
+      packages=['geoutils'],
+      install_requires=['rasterio', 'geopandas', 'pyproj'],
```
Comment (adehecq) [inline setup.py:37]: This should be consistent with the requirement.txt file, no?

Comment (atedstone) [reply setup.py:37]: Not necessarily: https://packaging.python.org/discussions/install-requires-vs-requirements/

File: .readthedocs.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: []
+
+python:
+  version: 3.7
+  install:
+    - requirements: requirements.txt
+    - method: pip
+      path: .
+  system_packages: false
```
Comment (jlandmann) [inline .readthedocs.yml:22]: Not even sure if that's "allowed" here, but if it wants a new line, we can give it a try ;-)

File: doc/conf.py [hunk 3]
```diff
@@ -93,160 +58,8 @@
...
-        pass
-
-    text = text_version if '+' not in GeoUtils.__version__ else text_dev
-
-    with open(filename, 'w') as f:
-        f.write(text)
-
-
-write_index()
+html_static_path = ['_static']
```
Comment (jlandmann) [inline doc/conf.py:65]: same here ;-)

File: geoutils/__init__.py [hunk 4]
```diff
@@ -4,6 +4,10 @@
 
 from . import georaster
 from . import geovector
+from os import path
+fn = path.join(path.dirname(__file__), 'version.py')
+print('Checking', fn)
+print('Does version.py exist?', path.exists(fn))
```
Comment (jlandmann) [inline geoutils/__init__.py:10]: Maybe this is not necessary anymore, but we can still keep it.

File: setup.py [hunk 5]
```diff
@@ -37,3 +25,18 @@ def write_version_py(filename=None):
 
 if write_version:
     write_version_py()
+
+
+setup(name='geoutils',
+      version=FULLVERSION,
+      description='',
+      url='',
```
Comment (jlandmann) [inline setup.py:33]: Add URL?

File: setup.py [hunk 6]
```diff
@@ -37,3 +25,18 @@ def write_version_py(filename=None):
...
+      author='The GlacioHack Team',
+      license='BSD-3',
+      packages=['geoutils'],
+      install_requires=['rasterio', 'geopandas', 'pyproj'],
+      extras_require={'rioxarray': ['rioxarray']},
+      scripts=[],
+      zip_safe=False)
+
+print('Does version.py exist?', path.exists(path.join(path.dirname(__file__), 'geoutils','version.py')))
```
Comment (jlandmann) [inline setup.py:42]: \n

**Review by jlandmann — APPROVED** [review]
Thanks for the great work! Looks goodto me! I just some really mini things...we should give it a try!

**Pull request merged**